modules/mod_acl_user_groups/support/acl_user_groups_checks.erl

```diff
@@ -661,31 +652,40 @@ can_rsc_for_collab_ugs(Id, Action, CollabId, CatId, UGs, Context) ->
     AllCollabId = m_rsc:rid(acl_collaboration_group, Context),
     lists:any(
         fun (GId) ->
-            case {
-                can_rsc_for_collab_ug(Id, Action, CollabId, CatId, GId, Context),
-                can_rsc_for_collab_ug(Id, Action, AllCollabId, CatId, GId, Context)
-            } of
-                {false, _AllCollabPermission} -> false;
-                {_CollabPermission, false} -> false;
-                {true, _AllCollabPermission} -> true;
-                {undefined, undefined} -> false;
-                {undefined, AllCollabPermission} -> AllCollabPermission
+            case can_rsc_for_collab_ug(Id, Action, CollabId, CatId, GId, Context) of
+                false ->
+                    % Deny rule: no permission to see the resource of the collab group itself
```

Comment:
Shouldn't this be:
Deny rule: no permission to see contents within this specific collab group
?